db/migrate/20250718074243_create_invite_decline_reasons.rb

```diff
@@ -0,0 +1,14 @@
+class CreateInviteDeclineReasons < ActiveRecord::Migration[7.0]
+  def change
+    create_table :invite_decline_reasons do |t|
+      t.string :reason
+      t.text :comment
+      t.string :status, default: 'draft'
+      t.references :invite, null: false, foreign_key: { to_table: :pool_invites }
```

Comment:
```suggestion
      t.references :invite, null: false, foreign_key: { to_table: :pool_invites, on_delete: :cascade }
```

Can we add this to the references. This replicates what rails does when you add `dependent: :destroy` in the models. But at the database level.

If we would do invite.delete, it would also delete the decline_resson now.

Reply:
@CatalinVoineag thanks, added! Also, to match existing pool models more closely, I've changed the name from invite_decline_reasons to pool_decline_reasons